lib/feature/home/task_list/ui/task_list_view.dart

```diff
@@ -31,6 +34,23 @@ class TaskListView extends StatelessWidget {
           title: Text(strings.get(StringKey.task_list_title)),
         ),
         body: _getBodyForState(context, state, strings),
+        drawer: Drawer(
+          child: ListView(
+            padding: EdgeInsets.zero,
+            children: <Widget>[
+              DrawerHeader(
+                decoration: BoxDecoration(color: ColorPalette.primary),
+                child: Text('Drawer Header', style: kWhiteTextStyle),
+              ),
+              ListTile(
+                title: Text('Logout'),
+                onTap: () async {
+                  await serviceLocator.get<UserManager>().logout();
```

Comment:
Trigger a Bloc event and do this action from the bloc. When we use Bloc, the view needs to be as simple and not-knowing as possible.